olive/passes/onnx/conversion.py

```diff
@@ -113,6 +113,36 @@ def _default_config(cls, accelerator_spec: AcceleratorSpec) -> Dict[str, PassCon
 
     def _run_for_config(
         self, model: PyTorchModelHandler, data_root: str, config: Dict[str, Any], output_model_path: str
+    ) -> Union[CompositeModelHandler, DistributedOnnxModelHandler, ONNXModelHandler]:
+        output_model = self._run_for_config_internal(model, data_root, config, output_model_path)
+
+        if isinstance(output_model, CompositeModelHandler):
```

Comment:
it seems this if/else check against different model type is already in pass.run? could we leverage that check and only add the save_* here for onnx model?

Reply:
Branching in Pass.run is conditioned on input model and here its conditioned on output model.